BeVolt/App/src/main.c

```diff
@@ -1344,4 +1347,32 @@ int main(void){
 	while(1){}
 }
 
+#elif defined DashboardMessage
```

Comment:
See the above comment.
```suggestion
#elif defined DASHBOARD_MSG
```